src/extensions/replay/sessionrecording.ts

```diff
@@ -980,7 +1009,11 @@ export class SessionRecording {
         }
 
         // Check if the URL matches any trigger patterns
-        this._checkUrlTrigger()
+        this._checkTriggerConditions()
+
+        if (this.status === 'paused' && !isRecordingPausedEvent(rawEvent)) {
```

Comment:
So I've added this to make sure this one event can go through even if the recording is paused, but... it seems like `onRRwebEmit` never gets called with this event?

Reply:
i ran it locally and these get called for me (in tests iirc there's no real rrweb so custom events don't work - could it be that)

<img width="549" alt="Screenshot 2024-10-31 at 11 43 20" src="https://github.com/user-attachments/assets/23e4db00-9591-41e8-a98d-cc8d686b9183">

i got lots of paused events... don't know if it's because i had breakpoints set or if i'm getting a paused on every mutation?